courses/models.py

```diff
@@ -0,0 +1,52 @@
+from django.core.urlresolvers import reverse
+from django.contrib.auth.models import User
+from django.db import models
+from jsonfield import JSONField
+from uuidfield import UUIDField
+
+class Course(models.Model):
+    """
+    An edX course which is purchaseable.
+    """
+    uuid = UUIDField(auto=True)
```

Comment:
I think since it's the only uuid in the model it makes sense to leave it this way.